astroplan/plots/plots.py

```diff
@@ -0,0 +1,216 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import matplotlib.pyplot as plt
+import numpy as np
+import astropy.units as u
+import random
+
+
+def plot_airmass(target, observer, time, ax=None, style_kwargs=None,
+                 dark_plot=False):
+    """
+    TODO:
+        1) Timezones?
+        2) Limit airmass <= 3.
+        3) Plot lines instead of points.
+        4) Legend should not be transparent.
+        5) dark_plot option
+
+    Makes an airmass vs. time plot.
+
+    If an ax object is passed in, plots an additional airmass plot on top.
+    Otherwise, creates a new ax object with an airmass plot.
+
+    When a `Time` object with a single instance in time is used (e.g., if
+    len(time) = 1), a new `Time` object is created internally.  This new object
```

Comment:
Also, the "sampling of times" are +/- 12 hours right? or is it 24 hours? Whatever it is, specify the size of the window.